Src/PerformanceCollector/Shared/Implementation/PerformanceCounterUtility.cs

```diff
@@ -33,7 +36,15 @@ internal static class PerformanceCounterUtility
             new Regex(
                 @"^\\(?<categoryName>[^(]+)(\((?<instanceName>[^)]+)\)){0,1}\\(?<counterName>[\s\S]+)$",
                 RegexOptions.Compiled);
-        
+
+        private static readonly Regex DisallowedCharsInReportAsRegex = new Regex(
+            @"[^a-zA-Z()/\\_. \t-]+",
+            RegexOptions.Compiled);
```

Comment:
Sanitizing report as is not working for perf collection. I temporarily disabled it. Please let me know if I am missing anything around this.